### ISSUE TITLE
Add support for postfix operators (required for the factorial operator)

### ISSUE THREAD

**Issue by alecrios** [alecrios/evaluator.js#2]
@jozsefsallai Turns out I missed a big issue with the factorial operator, so I reverted that bit while keeping the `factorial()` method.

The issue is that `!` is a postfix operator. At the moment, we only support prefix and infix operators, so we would need to make special modifications to the `convert()` function in `src/index.ts`.

Your `2 + 4!` test did pass, although, try running `4! + 2` and see how things break down.

If you're willing to look into how to add postfix support to the `convert()` function, that would be appreciated. I'll be looking into it as well, though maybe not immediately.

**Comment by jozsefsallai** [comment]
Yeah, I figured there would be issues with that at some point. I saw that it worked with infix, so I went with that because I couldn't find anything for postfix operators. Can't believe I missed trying stuff like `4! + 2` 🤦‍♂ Will try to look into this in my free time! Thanks for the merge!
